test/lib/resources/katello_cdn_test.rb

```diff
@@ -0,0 +1,59 @@
rratum\":0,\"package_group\":0,\"srpm\":0,\"module_stream\":0},\"mirroring_policy\":\"mirror_complete\",\"id\":56,\"name\":\"Red Hat Enterprise Linux 6 Server - Extended Life Cycle Support RPMs i386\",\"label\":\"Red_Hat_Enterprise_Linux_6_Server_-_Extended_Life_Cycle_Support_RPMs_i386\",\"description\":null,\"content_view_versions\":[],\"last_sync\":null,\"content_view\":{\"id\":3,\"name\":\"Default Organization View\"},\"content_view_version\":{\"id\":4,\"name\":\"Default Organization View 1.0\",\"content_view_id\":3},\"kt_environment\":{\"id\":3,\"name\":\"Library\"},\"content_type\":\"yum\",\"url\":\"https://cdn.redhat.com/content/els/rhel/server/6/6Server/i386/os\",\"arch\":\"i386\",\"os_versions\":null,\"content_id\":\"7396\",\"generic_remote_options\":null,\"major\":null,\"minor\":null,\"product\":{\"id\":292,\"cp_id\":\"204\",\"name\":\"Red Hat Enterprise Linux Server - Extended Life Cycle Support\",\"orphaned\":false,\"redhat\":true,\"sync_plan\":null},\"content_label\":\"rhel-6-server-els-rpms\",\"last_sync_words\":null},{\"backend_identifier\":\"816c4a3c-733f-459a-ba7d-75c4e3f7e34a\",\"relative_path\":\"test/Library/content/els/rhel/server/6/6Server/x86_64/os\",\"container_repository_name\":null,\"full_path\":\"https://sat-r220-07.lab.eng.rdu2.redhat.com/pulp/content/test/Library/content/els/rhel/server/6/6Server/x86_64/os/\",\"library_instance_id\":null,\"version_href\":\"/pulp/api/v3/repositories/rpm/rpm/2afb8e29-dc2d-484a-ba62-a895e176625b/versions/1/\",\"remote_href\":\"/pulp/api/v3/remotes/rpm/rpm/a8bc43b5-6c18-46fe-9ca6-730382c2eefa/\",\"publication_href\":\"/pulp/api/v3/publications/rpm/rpm/b6949e34-458a-4d0d-b29d-6455dc7a2373/\",\"content_counts\":{\"rpm\":21295,\"erratum\":4459,\"package_group\":202,\"srpm\":0,\"module_stream\":0},\"mirroring_policy\":\"mirror_complete\",\"id\":51,\"name\":\"Red Hat Enterprise Linux 6 Server - Extended Life Cycle Support RPMs x86_64\",\"label\":\"Red_Hat_Enterprise_Linux_6_Server_-_Extended_Life_Cycle_Support_RPMs_x86_64\",\"description\":null,\"content_view_versions\":[],\"last_sync\":{\"id\":\"ddc851af-f130-4a96-84ad-c35725bb9ca2\",\"username\":null,\"started_at\":\"2022-03-01 19:23:30 UTC\",\"ended_at\":\"2022-03-01 19:45:09 UTC\",\"state\":\"stopped\",\"result\":\"success\",\"progress\":1.0},\"content_view\":{\"id\":3,\"name\":\"Default Organization View\"},\"content_view_version\":{\"id\":4,\"name\":\"Default Organization View 1.0\",\"content_view_id\":3},\"kt_environment\":{\"id\":3,\"name\":\"Library\"},\"content_type\":\"yum\",\"url\":\"https://cdn.redhat.com/content/els/rhel/server/6/6Server/x86_64/os\",\"arch\":\"x86_64\",\"os_versions\":null,\"content_id\":\"7396\",\"generic_remote_options\":null,\"major\":null,\"minor\":null,\"product\":{\"id\":292,\"cp_id\":\"204\",\"name\":\"Red Hat Enterprise Linux Server - Extended Life Cycle Support\",\"orphaned\":false,\"redhat\":true,\"sync_plan\":null},\"content_label\":\"rhel-6-server-els-rpms\",\"last_sync_words\":\"about 22 hours\"}]}\n"
```

Comment:
This may be better off in a separate file. Use the idea here  -> https://github.com/Katello/katello/blob/master/test/services/katello/pulp3/content_test.rb#L15